src/polyglot-notebooks-vscode-common/src/extension.ts

```diff
@@ -140,7 +140,8 @@ export async function activate(context: vscode.ExtensionContext) {
 
         const workspaceFolderUris = vscode.workspace.workspaceFolders?.map(folder => folder.uri) || [];
         const workingDirectory = getWorkingDirectoryForNotebook(notebookUri, workspaceFolderUris, fallbackWorkingDirectory);
-        const environmentVariables = polyglotConfig.get<{ [key: string]: string }>('kernelEnvironmentVariables');
+        const environmentVariables = { ...polyglotConfig.get<{ [key: string]: string }>('kernelEnvironmentVariables'), 'DOTNET_CLI_CULTURE': getCurrentCulture(), 'DOTNET_CLI_UI_LANGUAGE': getCurrentCulture() };
+
```

Comment:
A similar change will have to be made in a few places, unfortunately:

<img width="709" alt="image" src="https://github.com/dotnet/interactive/assets/547415/c4ad860d-f0a7-4a84-8b44-8065cd2b04e5">

@colombod We should discuss whether we can reduce some duplication here.

Reply:
There is only one place.

Extension.ts is a link and it appears at these different places